src/v2/views/RegisterView/index.tsx

```diff
@@ -64,6 +64,7 @@ function RegisterView() {
         address={key?.address}
         onSubmit={onSubmit}
         useActivitionKey
+        useLink={"https://nine-chronicles.com/start?step=2"}
```

Comment:
I also guess that current URL(`"https://nine-chronicles.com/start?step=2"`) might be changed for various reasons. (In particular, query strings such as `step=2` often change according to the specifications of the service.) in that case, we need to update the whole binary... 😢 

- How about using config instead hard coded value?
- Or, leveraging indirection URL seems an option too.

cc @Gilhwan @rareboe @minmboy